board/microchip/polarberry/post-image.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+#Create FAT linux Image (linux kernel  + rootfs + DTS)
+cp -R output/images ${BR2_EXTERNAL_MCHP_PATH}/board/microchip/polarberry
+/usr/bin/mkimage -A riscv -O linux -T flat_dt -f ${BR2_EXTERNAL_MCHP_PATH}/board/microchip/polarberry/fitimage_polarberry.its output/images/fitimage_polarberry.itb
+
+#Copy UBOOT adopted image to the TFTP server directory
+#sudo cp output/images/payload.bin /srv/tftp
+
+#Copy Linux FTD image to the TFTP server directory
+#sudo cp output/images/fitimage_polarberry.itb /srv/tftp
+
+#clean up
```

Comment:
Please remove lines 7-13. They should not be here. This behaviour does not belong in a Buildroot post image script and should be done outside of the Buildroot build if required. If you need to do something like this, please add them to the (relevant) existing shell script files in the boards directory.